sylvia-derive/src/querier.rs

```diff
@@ -157,9 +157,7 @@ where
         let querier = trait_module
             .map(|module| quote! { #module ::sv::Querier })
             .unwrap_or_else(|| quote! { sv::Querier });
-        let bound_querier = contract_module
-            .map(|module| quote! { #module ::sv::BoundQuerier})
-            .unwrap_or_else(|| quote! { sv::BoundQuerier });
+        let bound_querier = quote! { #contract_module ::sv::BoundQuerier };
```

Comment:
Same. as `contract_module` is not an option anymore we can just remove this quote and use these values directly in the main quote

Reply:
Done